source/libraries/HelperFunctions.ts

```diff
@@ -74,14 +75,15 @@ export async function initializeTestRpcClientOptions(ethHttpProviderPort: number
     return { gasLimit: GAS_BLOCK_AMOUNT, accounts: accountOptions };
 }
 
-async function sleep(milliseconds: number): Promise<object> {
-    return new Promise((resolve, reject) => setTimeout(resolve, milliseconds));
+export async function sleep(milliseconds: number): Promise<void> {
```

Comment:
Do we actually need to export this?  I didn't see where it's being used outside of this file.

Reply:
In the past I have found that having an async sleep can be really useful when debugging, so even though we may only use it directly here, having it exported can be really useful.  Since this is in a `HelperFunctions` file I made an exception to my usual personal rule of "only export things you actually need to export", since it is so useful for debugging.